Extensions/xkit_patches.js

```diff
@@ -1670,7 +1670,8 @@ XKit.tools.getParameterByName = function(name){
 					XKit.tools.add_css("." + class_name + ":after {" +
 							" background-image: url('" + icon + "') !important;" +
 							" background-size: auto auto !important;" +
-							" margin-top: -9px !important; " +
+							" margin-top: -7px !important; " +
+							" opacity: 1 !important; " +
```

Comment:
@nightpool  `opacity: 1 !important` occurs a ton of other places in the file, I'm inclined to allow this modification as it is locally consistent